plugins/lighthouse-backend/src/config.ts

```diff
@@ -13,56 +13,66 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { LoggerService } from '@backstage/backend-plugin-api';
+import {
+  TaskScheduleDefinition,
+  readTaskScheduleDefinitionFromConfig,
+} from '@backstage/backend-tasks';
+import { Config, readDurationFromConfig } from '@backstage/config';
+import { HumanDuration } from '@backstage/types';
 
-import { Config } from '@backstage/config';
-import { HumanDuration as HumanDuration } from '@backstage/types';
-
-export interface LighthouseAuditScheduleConfig {
-  schedule: HumanDuration;
-  timeout: HumanDuration;
-  auditDetail: HumanDuration;
-}
-
-/** @public */
-export type LighthouseAuditSchedule = {
-  getSchedule: () => HumanDuration;
-  getTimeout: () => HumanDuration;
+type Options = {
+  logger: LoggerService;
 };
 
 /** @public */
-export class LighthouseAuditScheduleImpl implements LighthouseAuditSchedule {
-  static fromConfig(config: Config) {
-    const lighthouse = config.getOptionalConfig('lighthouse');
+export class LighthouseAuditScheduleImpl implements TaskScheduleDefinition {
+  /**
+   * Looks at the `lighthouse.schedule` section in the application configuration
+   * and returns a TaskScheduleDefinition.
+   * Defaults to `{ frequency: { days: 1 }, timeout: {}, initialDelay: { minutes; 15 } }`
+   *
+   * @returns a TaskScheduleDefinition
+   */
+  static fromConfig(config: Config, options: Options): TaskScheduleDefinition {
+    const { logger } = options;
+
+    let lighthouse: TaskScheduleDefinition = {
+      frequency: { days: 1 },
+      timeout: {},
```

Comment:
again, this actually is not possible since it's an invalid duration and schedules MUST have a timeout (it's not optional) - set this to like `{ minutes: 10 }` or whatever makes sense.